src/app/screens/LNURLPay.tsx

```diff
@@ -40,11 +40,11 @@ function LNURLPay(props: Props) {
         JSON.parse(searchParams.get("origin") as string)) ||
       getOriginData()
   );
-  const [valueMSat, setValueMSat] = useState<number | undefined>(
-    details?.minSendable
+  const [valueSat, setValueSat] = useState(
+    (details?.minSendable && (details?.minSendable / 1000).toString()) || ""
```

Comment:
also here isn't `details?.minSendable` something from the JSON response. what if it is a string?

Reply:
According to the spec it should be a number?

Reply:
it should but it is still an API response and people to strange things there :)